src/view/components/common/push-publish/add-to-bundle-dialog-component.ts

```diff
@@ -3,6 +3,9 @@ import {CORE_DIRECTIVES} from "angular2/common";
 import {ModalDialogComponent} from "../modal-dialog/dialog-component";
 import {Dropdown, InputOption} from "../../semantic/modules/dropdown/dropdown";
 import {IBundle} from "../../../../api/services/bundle-service";
+import {ViewChild} from "angular2/core";
+
+var $ = window['$']
```

Comment:
Yes, var $ = window['$'] is unused, I added it to test something but later I forgot remove it